nss-day-cohort-31/celebrity-tribute-picture-perfect
# Personal-page

Added Nav bar
added unordered list containing personal info
Added list of immediate family members including spouse parents and sister
Added 3 images one of his father, mother , and spouse
Added H tags to the images 
Added a tag to Fathers picture to link to his wikipedia

Steps to test:
1. 'git fetch --all'
2. git checkout js-personal
3. Run server
4. Check list as well as images
